packages/flutter/lib/src/cassowary/equation_member.dart

```diff
@@ -5,25 +5,64 @@
 import 'expression.dart';
 import 'constraint.dart';
 
-/// Base class for the various parts of cassowary equations.
+/// Represents a member that can be used to construct an expression that may be
+/// used to create a constraint. This is to facilitate the easy creation of
+/// constraints. The use of the operator overloads is completely optional and
+/// is only meant as a convenience. The constraint expressions can be created
+/// by manually creating instance of constraint variables, then terms and
+/// combining those to create expression.
 abstract class EquationMember {
+  /// The representation of this member after it is hoisted to be an
+  /// expression.
   Expression asExpression();
 
+  /// If this member is a constant.
   bool get isConstant;
 
+  /// The current constant value of this member.
   double get value;
 
+  /// Create a constraint by using this member as the left hand side expression
```

Comment:
(this comment applies to all operators here)

"Creates" rather than "Create".

Consider following the pattern established at http://docs.flutter.io/flutter/rendering/Point-class.html